mpire/pool.py

```diff
@@ -431,6 +432,7 @@ def map(self, func: Callable, iterable_of_args: Union[Sized, Iterable], iterable
             MPIRE will raise a ``TimeoutError``. Use ``None`` to disable (default).
         :param progress_bar_options: Dictionary containing keyword arguments to pass to the ``tqdm`` progress bar. See
             ``tqdm.tqdm()`` for details. The arguments ``total`` and ``leave`` will be overwritten by MPIRE.
+        :param progress_bar_backend: The progress bar backend to use. Can be one of ``None``, ``std``, or ``notebook``
```

Comment:
Nitpick but I guess this should be ``'std'`` and ``'notebook'``?

Reply:
Thnx for nitpicking! Please be as thorough as you can :D 

And you mean that it should be with quotes, right? In that case, absolutely :)

Reply:
Yes, github formatting removed the `` :)